postgres/macros.jinja

```diff
@@ -18,7 +18,8 @@
   {%- if 'ensure' in kwarg %}
     {%- set ensure = kwarg.pop('ensure') %}
   {%- endif %}
-  {%- if 'user' not in kwarg and state != 'postgres_schema' %}
+  {%- set user_available = False if (state == 'postgres_schema' and grains.saltversioninfo < [2018, 3]) else True %}
```

Comment:
I think there is no need for such complex ternary operator, it could be done simply as:
```suggestion
  {%- set user_available = not (state == 'postgres_schema' and grains.saltversioninfo < [2018, 3]) %}
```

Reply:
@vutny Applied the suggestion, thanks.